test/tests/functional/pbs_verify_log_output.py

```diff
@@ -113,31 +112,30 @@ def test_if_info_add(self):
         """
         Test for interface info presence in log files
         """
-        names = "["
-        for name in [x[0] for x in self.all_interfaces()]:
-            names += "(" + name + ")"
-        names += "]"
+        interfaceGenerator = self.all_interfaces()
+        for ifname in interfaceGenerator:
+            name = "[(" + ifname + ")]"
 
-        log_val = "[(ipv4)(ipv6)(ipv4/ipv6)] [(interface: )]" + names
+            log_val = "[( interface: )]" + name
```

Comment:
From what I know, () represent groups of string that need to be matched together. Here, it might seem a little slightly weird, but I tried "(((ipv4)|(ipv6)|(ipv4\/ipv6)) (interface) (lo))" and it just wouldn't match in the server logs - it matched in the mom and sched logs, despite the same value being present in all the three logs. The value in the logs is: "ipv4 interface lo", which should match with the regex I tried earlier.

I even verified that it should work with an online regex tool (http://regexr.com/). But with PTL, no dice.